data/campaigns/Descent_Into_Darkness/scenarios/09_Descent_into_Darkness.cfg

```diff
@@ -34,7 +34,7 @@
             {STORYTXT_BACKGROUND book.jpg}
         [/part]
         [part]
-            story= _ "So reads the book that Malin has reclaimed. <i>“The spells of necromancy can free the spirit from the limitations of the flesh, but only once the soul has been unbound from the body. The necromancer must make the necessary incantations with his dying breaths, then conquer his own spirit much the same way he binds the spirits of others. Because he retains his own will, however, the lich can call upon the awesome powers of the spirit world.”</i>"
+            story= _ "So reads the book that Malin has reclaimed. <i>“The spells of necromancy can free the spirit from the limitations of the flesh, but only once the soul has been unbound from the body. The necromancer must make the necessary incantations with his dying breaths, then conquer his own spirit in much the same way he binds the spirits of others. Because he retains his own will, however, the lich can call upon the awesome powers of the spirit world.”</i>"
```

Comment:
Caught a necromancer trying to sneak in with the moon-language changes!

Reply:
wait this is something that `luacheck` caught?

Reply:
What? This is something I did in 3922d04b446f9f4bb06441ca4644adf25b35fc69, to revert a mistake in 19a42e3ead717dc1808162168585abe8755df4a4. Maybe Pent's branch isn't forked from current master?

Reply:
I'm really confused with what git thinks it's doing right now.

Reply:
Should be good now.

Reply:
Yep.